### PR TITLE
Don't require `:plugins` `Gemfile` group when running `hanami new`

### PR DESCRIPTION
**Pull request by jodosha**
Fixes: https://github.com/hanami/hanami/issues/838

### CODE REVIEW

File: bin/hanami [hunk 1]
```diff
@@ -2,5 +2,5 @@
 require 'bundler'
 require 'hanami/cli/commands'
 
-Bundler.require(:plugins)
+Bundler.require(:plugins) if File.exist?(ENV["BUNDLE_GEMFILE"] || "Gemfile")
```
Comment (davydovanton) [inline bin/hanami]: WDYT maybe is better to wrap this line by method `load_plugins`?

Comment (jodosha) [reply bin/hanami]: @davydovanton I considered that too, but I'm not sure yet about that API. `load_plugins` looks like a complete plugins system, which is not there yet.

Comment (davydovanton) [reply bin/hanami]: @jodosha agree. We can fix it in future (of course, if we need it)

**Comment by cllns** [comment]
Not necessary right now, but we shouldn't rely on `Gemfile`, hard-coded, since bundler already supports using `gems.rb` instead and apparently that will be the default in the future. (Will have to other changes in Hanami too, most likely)

**Comment by jodosha** [comment]
@cllns My idea is to make Hanami 2.0 to support `gems.rb`.

**Pull request merged**